content/posts/help-videos.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Help videos now available"
+date: 2023-10-31
+version: ["hint v3.3.0"]
+pull_request: https://github.com/mrc-ide/hint/pull/877
+tags: ["Feature"]
+---
+
+2 years ago we created some instruction videos about how to use Naomi. We've now added a link from the "Online support" menu in the header to the videos. We've plans to update and expand these video tutorials going forwards so stay tuned for updates!
```

Comment:
I think :laughing: also would be nice to just have a link to the youtube channel here too no?
```suggestion
2 years ago we created some instruction videos about how to use Naomi. We've now added a link from the "Online support" menu in the header to the videos. We've plans to update and expand these video tutorials going forward so stay tuned for updates!
```

Reply:
Thanks, I've added that link now